src/components/refactor/dialogs/line-to-attach-or-split-form/line-to-attach-or-split-utils.js

```diff
@@ -41,3 +41,20 @@ export const getLineToAttachOrSplitFormData = ({
         }),
     };
 };
+
+export const buildNewBusbarSections = (
+    equipmentId,
+    sectionCount,
+    busbarCount
+) => {
+    const newBusbarSections = [];
+    for (let i = 0; i < sectionCount; i++) {
+        for (let j = 0; j < busbarCount; j++) {
+            newBusbarSections.push({
+                id: equipmentId + '_' + (j + 1) + '_' + (i + 1),
+                name: '',
+            });
+        }
+    }
+    return newBusbarSections.sort((a, b) => a?.id?.localeCompare(b?.id));
```

Comment:
if you loop first on busbarCount and then on sectionCount, there is not need to sort, no ?